composer.json

```diff
@@ -15,19 +15,29 @@
         "drupal/address": "~1.2",
         "drupal/admin_toolbar": "~1.20",
         "drupal/allowed_formats": "~1.1",
+        "drupal/better_normalizers": "1.x-dev",
```

Comment:
Lets lock this to a hash

Reply:
i dont know why it decided to use that version. i can switch it to the beta